pymeasure/instruments/wentworthlabs/s200.py

```diff
@@ -0,0 +1,548 @@
+        "PSTH", "GTTH %d",
+        "Control the rotation of the chuck to the Theta-axis position in millidegrees "
+        "specified in the parameter",
+        # AWP compatible: No
+        validator=strict_range,
+        values=[-359999, 359999],
+        check_set_errors=True,
+        get_process=lambda r:
+        float(r.replace("PSTH ", ""))
+    )
+
+    z_overtravel = Instrument.control(
+        "RZIM", "WZIM %d",
+        "Control the z-axis overtravel (in um) of the chuck of the probe station",
+        # AWP compatible: No
+        validator=strict_range,
+        values=Z_OVERTRAVEL_VALID_RANGE,
+        check_set_errors=True,
+        get_process=lambda r:
+        float(r.replace("RZIM ", ""))
+    )
+
+    z_grosslift = Instrument.control(
+        "RKGM", "WKGM %d",
+        "Control the z-axis gross lift (in um) of the chuck of the probe station",
+        # AWP compatible: No
+        validator=strict_range,
+        values=Z_GROSSLIFT_VALID_RANGE,
+        check_set_errors=True,
+        get_process=lambda r:
+        float(r.replace("RKGM ", ""))
+    )
+
+    z_finelift = Instrument.control(
+        "RKFM", "WKFM %d",
+        "Control the z-axis fine lift (in um) of the chuck of the probe station",
+        # AWP compatible: No
+        validator=strict_range,
+        values=Z_FINELIFT_VALID_RANGE,
+        check_set_errors=True,
+        get_process=lambda r:
+        float(r.replace("RKFM ", ""))
+    )
+
+    indexing_mode = Instrument.setting(
+        "%s",
+        "Control the indexing mode."
+        "If True, Enters indexing mode and moves to the first die to be probed, or moves to the"
+        " next die to be probed if already in indexing mode. Indexing mode is exiting"
+        " using the NXF command (above). Pay attention. When in indexing mode, "
+        " next_die property will be not recognized as a command and will get the state of the table"
+        " out of communication."
+        " All subsequent commands will be ignored and thus we will re-enter in remote mode.",
+        # AWP compatible: Yes
+        validator=strict_discrete_set,
+        values={True: 'NXT', False: 'NXF'},
+        map_values=True,
+        check_set_errors=True
+    )
+
+    next_die = Instrument.setting(
+        "NX%s",
+        "Control the movement of the chuck to the next die",
+        # If probe_table.next_die = "D" then it will go below the current die
+        # If probe_table.next_die = "U" then it will go above the current die
+        # If probe_table.next_die = "L" then it will go to the left of the current die
+        # If probe_table.next_die = "R" then it will go to the right of the current die
+        # AWP compatible: Yes
+        check_set_errors=True,
+        validator=strict_discrete_set,
+        values=['D', 'U', 'L', 'R']
+    )
+
+    status_byte = Instrument.measurement(
+        "STA",
+        "Measure the status byte of the instrument",
+        get_process=lambda r:
+        r
```

Comment:
All of these `lambda r: r` can be removed, as they are the same as the default `get_process` lambda.